app/models/app_setting.rb

```diff
@@ -17,15 +17,7 @@ class AppSetting < ApplicationRecord
   after_commit :apply_settings, on: :update
 
   def self.instance
-    first_or_create!(singleton_guard: 0,
-                     github_user_name: 'MyUserName',
-                     github_user_email: 'example@email.com',
-                     git_repository_name: 'repository',
-                     git_repository_url: 'https://github.com/hpi-swt2/vm-portal.git',
-                     puppet_init_path: '/',
-                     puppet_nodes_path: '/Node',
-                     puppet_classes_path: '/Name',
-                     vm_archivation_timeout: 3) # in days
+    find(1)
```

Comment:
Without seed you cannot log in (the application already wants to use some values of AppSetting at this moment). Therefore I changed it so that you get a message to seed instead of the button to log in. Because of that the admin have to seed the application before the first use and we have a valid state.
A side effect  is that find(1) should always be ok

(first_or_create!() would not be enough without give it almost all the values that are in the seeds.rb)

Reply:
:+1: